<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBoxTermCredit">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>72</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
  border: 1px solid gray;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>18250</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDifferential">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Differential</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinLoanBody">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
  border: 1px solid gray;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>999999999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTotalPaymentResult">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>320</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalPaymentsCredit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
 background-color: rgb(255, 201, 157);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>  Total Payment</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAnnuitete">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Annuitete</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMonthlyPaymentResult">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>196</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalcCredit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>703</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 145, 200);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Caclulate</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMonthlyPayment">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>196</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
 background-color: rgb(255, 201, 157);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>  Monthly Payment</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoanOverpay">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>258</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
 background-color: rgb(255, 201, 157);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>  Loan Overpay</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOverpayResult">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>258</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPeriodInterest">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>134</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(155, 181, 137);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>  Period interest</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTermCredit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>72</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(155, 181, 137);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>  Term</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinInterestCredit">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>134</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
  border: 1px solid gray;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoanBodyCredit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(155, 181, 137);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>  Loan Body</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>doubleSpinLoanBody</tabstop>
  <tabstop>spinBoxTermCredit</tabstop>
  <tabstop>doubleSpinInterestCredit</tabstop>
  <tabstop>lineEditMonthlyPaymentResult</tabstop>
  <tabstop>lineEditOverpayResult</tabstop>
  <tabstop>lineEditTotalPaymentResult</tabstop>
  <tabstop>radioButtonAnnuitete</tabstop>
  <tabstop>radioButtonDifferential</tabstop>
  <tabstop>CalcCredit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
